src/components/VmDetails/cards/UtilizationCard/CpuCharts.js

```diff
@@ -48,33 +48,39 @@ const CpuCharts = ({ cpuStats, isRunning, id }) => {
           </UtilizationCardDetails>
 
           <DonutChart
-            id={`${id}-donut-chart`}
-            data={{
-              columns: [
-                [msg.utilizationCardLegendUsedP(), cpuUsed],
-                [msg.utilizationCardLegendAvailableP(), cpuAvailable],
-              ],
-              order: null,
-            }}
-            title={{
-              primary: `${cpuUsed}`, // NOTE: String else 0 is truthy false and doesn't render proper
-              secondary: msg.utilizationCardLegendUsedP(),
-            }}
-            tooltip={{
-              show: true,
-              contents: patternfly.pfDonutTooltipContents,
-            }}
+            data={[
+              {
+                x: msg.utilizationCardLegendUsedP(),
+                y: cpuUsed,
+                label: `${msg.utilizationCardLegendUsed()} - ${cpuUsed}%`,
+              },
+              {
+                x: msg.utilizationCardLegendAvailableP(),
+                y: cpuAvailable,
+                label: `${msg.utilizationCardAvailable()} - ${cpuAvailable}%`,
+              },
+            ]}
+            subTitle={msg.utilizationCardLegendUsedP()}
+            title={`${cpuUsed}`}
           />
 
           { history.length === 0 && <NoHistoricData id={`${id}-no-historic-data`} /> }
           { history.length > 0 &&
-            <SparklineChart
-              id={`${id}-line-chart`}
-              data={{
-                columns: [['%', ...history]],
-                type: 'area',
-              }}
-            />
+            <ChartGroup
```

Comment:
> The padding that is needed between the chart's box and the chart itself looks to be extra padding so the tooltip renders without getting clipped right? it would be nice to have a way for the tooltip to not have to render in the svg box, so it won't get clipped, and the chart itself can be made bigger (e.g. `padding={0}`)

@thaorell's comment fixes it, and that was integrated :+1: